docs/signature/README.md

```diff
@@ -0,0 +1,228 @@
+# Notary V2 Signature Specification
+
+This section defines the signature file, which is in JSON format with no whitespaces. Its JSON schema is available at [schema.json](schema.json).
+
+## Signature Goals
+
+- Offline signature creation
+- Persistance within an [OCI Artifact][oci-artifacts] enabled, [distribution-spec][distribution-spec] based registry
+- Artifact and signature copying within and across [OCI Artifact][oci-artifacts] enabled, [distribution-spec][distribution-spec] based registries
+- Support public registry acquisition of content - where the public registry may host certified content as well as public, non-certified content
+- Support private registries, where public content may be copied to, and new content originated within
+- Air-gapped environments, where the originating registry of content is not accessable
+- Multiple signatures per artifact, enabling the originating vendor signature, public registry certification and user/environment signatures
+- Maintain the original artifact digest and collection of associated tags, supporting dev/ops deployment definitions
+
+## Signature
+
+A Notary v2 signature is clear-signed signature of manifest metadata, including but not limited to
```

Comment:
This is good, it's better than signing simply tags instead of higher-level manifests or indices, which is a problem that the TUF team and I have been highlighting as one of the problems with notary-v1.